Update "New To Matic" to include examples on how to query data from Polygon blockchain. (#1154)
maticnetwork/matic-docs: docs/home/new-to-matic.md

* [Ethereum + Polygon with PoS bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/pos/getting-started)
* [Ethereum + Polygon with Plasma bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/plasma/getting-started)

> ## Query the Blockchain

Reviewer: ```suggestion
## Query the blockchain

```

> 80% of web3 developers don't need to write a single smart contract - instead, most interaction with the blockchain involves reading its current state.

Reviewer: ```suggestion
Most blockchain interactions involve reading its state.
```

> If you're in that 80%, here are some examples on how to make basic requests to the blockchain.

Reviewer: ```suggestion
Alchemy offers a reference guide on how to make basic requests to the blockchain.
Check out their guide on [how  to query Polygon](https://docs.alchemy.com/reference/polygon-sdk-examples).
```

* [Ethereum + Polygon with PoS bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/pos/getting-started)
* [Ethereum + Polygon with Plasma bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/plasma/getting-started)

## Query the Blockchain
Most blockchain interactions involve reading its state.

Alchemy offers a reference guide on how to make basic requests to the blockchain.
Check out their guide on [how  to query Polygon](https://docs.alchemy.com/reference/polygon-sdk-examples).

> * [How to Query the Polygon Blockchain](https://docs.alchemy.com/reference/polygon-sdk-examples)

Reviewer: ```suggestion
```
